### PR TITLE
bump to alpha2

### PR DESCRIPTION
**Pull request by wsnipex**
bump to alpha2, remove addons-no-startupenable.patch with new build option

### CODE REVIEW

**Comment by razzeee** [comment]
Build seems fine from a quick check

File: tv.kodi.Kodi.yml [hunk 1]
```diff
@@ -116,39 +118,42 @@ modules:
...
         dest: build/download/
-        dest-filename: addon-list.txt
       - type: file
         path: tools/patch-addon-manifest.sh
         dest: build/download/
       - type: shell
         commands:
           - build/download/patch-addon-manifest.sh build/download/addon-list.txt
-      - type: patch
-        path: addons-no-startupenable.patch
```
Comment (razzeee) [inline tv.kodi.Kodi.yml:133]: I guess that file should be deleted?

File: tv.kodi.Kodi.yml [hunk 2]
```diff
@@ -116,39 +118,42 @@ modules:
...
+          - git add system/addon-manifest.xml
+          - git commit system/addon-manifest.xml -m "[patch] make addons optional"
```
Comment (razzeee) [inline tv.kodi.Kodi.yml:138]: Why are we doing this?

Comment (wsnipex) [reply tv.kodi.Kodi.yml:138]: I'd like to get rid of the "dirty" in version info, but looks like it didn't work

**Review by razzeee — APPROVED** [review]

**Pull request merged**